tests/system/providers/google/cloud/automl/example_automl_nl_text_classification.py

```diff
@@ -114,7 +112,7 @@
         import_configs=DATA_CONFIG,
     )
 
-    # [START howto_operator_automl_create_model]
+    # [START how_to_cloud_create_text_classification_training_job_operator]
```

Comment:
nitpick: Looks like `howto` is the convention of these example DAGs. Not sure whether we want to change it. If so, we probably should change all (might not be in this PR)

Reply:
I have changed this to be more clear what this example is about. Maybe we should change it back, but there are a lot of automl examples for text, translate, video and vision that are named the same way, like `howto_operator_automl_create_model`.
@potiuk What do you think about this?

Reply:
I think whet @Lee-W proposed here is to just change `how_to` to `howto` (but leave the rest of the change. 

And I think yes - it makes sense for consistency.